package.json

```diff
@@ -54,6 +54,11 @@
     "@babel/preset-typescript": "^7.12.7",
     "@babel/register": "^7.0.0",
     "@rollup/plugin-node-resolve": "^13.0.0",
+    "@semantic-release/changelog": "^5.0.1",
+    "@semantic-release/commit-analyzer": "^8.0.1",
```

Comment:
I think all of these packages are part of the `semantic-release` module. But there is nothing bad with defining them explicitly here. For reference: 

https://github.com/contentful/contentful-management.js/blob/master/package.json#L125
https://github.com/contentful/contentful-management.js/blob/master/package.json#L77